augur/application/db/models/augur_data.py

```diff
@@ -3600,4 +3601,4 @@ class RepoClone(Base):
     count_clones = Column(BigInteger)
     clone_data_timestamp = Column(TIMESTAMP(precision=6))
 
-    repo = relationship("Repo")
\ No newline at end of file
+    repo = relationship("Repo")
```

Comment:
sorry about that bit, here my Python checker is just removing a trailing newline.